bin/drop_sample_annot.py

```diff
@@ -148,6 +156,9 @@ def parse_args(argv=None):
     parser.add_argument(
         "--strandedness", type=str, nargs="+", help="strandedness of RNA", required=True
     )
+    parser.add_argument(
+        "--sex", type=str, nargs="+", help="Sex of samples", required=True
+    )
```

Comment:
I think that you will have to make some modifications within the python script and module so that it works when sex is not given

Reply:
Yes, maybe. Not sure how the default value applies. I'll do a test-run and check.

Reply:
I tested running it on a samplesheet without sex column. Here is how the channel look when `.view()`:

```
[[case:ID_case, id:ID_id1, sample:ID, strandedness:reverse, sex:NA, single_end:false, fq_pairs:1], [/fs2/jakob/data/tomtedata/ID_S7_R1_001.fastq.gz, /fs2/jakob/data/tomtedata/ID_S7_R2_001.fastq.gz]]
```

So it looks like the column gets added, which would work with the `drop_sample_annot.py` script further changes. Or what do you think?

Reply:
Likely yes, I will give it a try locally

Reply:
Sounds great, thanks!

Reply:
Looks OK in the channel at least on a quick test:

```
[[case:case, id:s1_id2, sample:s1, strandedness:reverse, sex:NA, single_end:false, fq_pairs:1], [/fs2/jakob/data/tomtedata/ID_S7_R1_001.fastq.gz, /fs2/jakob/data/tomtedata/ID_S7_R2_001.fastq.gz]]
[[case:case, id:s2_id2, sample:s2, strandedness:reverse, sex:NA, single_end:false, fq_pairs:1], [/fs2/jakob/data/tomtedata/ID_S7_R1_001.fastq.gz, /fs2/jakob/data/tomtedata/ID_S7_R2_001.fastq.gz]]
[[case:case, id:s3_id2, sample:s3, strandedness:reverse, sex:NA, single_end:false, fq_pairs:1], [/fs2/jakob/data/tomtedata/ID_S7_R1_001.fastq.gz, /fs2/jakob/data/tomtedata/ID_S7_R2_001.fastq.gz]]
```

But anyway great if you want to test further to ensure it works OK 👌

Reply:
You are absolutely right, it works like a charm 😄